JS_2/task3.js

```diff
@@ -0,0 +1,26 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+const age = +prompt("Enter your age, please");
+const age_2 = 18;
+const age_3 = 60;
+switch (true) {
+  case 0:
+  case -1:
+  case "my age":
+    console.log("Invalid age");
```

Comment:
Probably you need add some comparisions to get in this case

Why we check exactly this cases `0, -1, 'my age'` ? I will input `-2` or left it empty, for example
`isNaN` can help you achieve the correct result in `if...else` statement before comparing the ages in `switch`

Reply:
thanks for advice)